### PR TITLE
Add psutil

### PR DESCRIPTION
**Pull request by jakirkham**
Add a recipe to build psutil. Used `conda skeleton pypi` to generate the recipe. Then cleaned it up for addition here.

### CODE REVIEW

**Comment by jakirkham** [comment]
Oops, thought I had that.

**Comment by jakirkham** [comment]
Ouch, Python 2.7 fails to build on Windows. Will need to investigate.

File: recipes/psutil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - psutil.tests
+
+about:
+  home: https://github.com/giampaolo/psutil
+  license: BSD 3-Clause
+  summary: A cross-platform process and system utilities module for Python
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```
Comment (pelson) [inline recipes/psutil/meta.yaml]: Happy to be added here.

Comment (jakirkham) [inline recipes/psutil/meta.yaml]: Added. :smile:

**Comment by pelson** [comment]
Timing out on Windows (and a huge build log, so I haven't been able to scroll to the bottom yet).

**Comment by jakirkham** [comment]
Seems there is an issue ( https://github.com/conda/conda-build/issues/830 ) open about the Windows problems. Will skip Windows for now and feedstock. We can handle the Windows issues after.

**Comment by jakirkham** [comment]
Alright, I'm going to clear this one. Feedback welcome at the feedstock.

**Comment by pelson** [comment]
:+1: - thanks @jakirkham.

**Comment by jakirkham** [comment]
@giampaolo, we have added `psutil` to [conda-forge]( https://conda-forge.github.io/ ). This is the community packaging system for use with [`conda`]( http://conda.pydata.org/docs/ ). Packages built here are built with CI and deployed to the [conda-forge channel]( https://anaconda.org/conda-forge/ ) automatically after committing on `master` of a feedstock (a repo with the recipe plus automatically maintained CI scripts). Eventually, packages submitted here will be included in the `defaults` channel after some auditing process as [mentioned]( https://github.com/conda-forge/conda-forge.github.io/issues/22#issuecomment-197117472 ). If there is anything here that you see is off or needs to be fixed, please let me know. If you would like to have access to the [psutil feedstock/recipe]( http://github.com/conda-forge/psutil-feedstock/ ), I would happily add you to the maintainers list so you can have commit privileges for this recipe. This would be useful for updating the version for instance. Thanks again for working on such a useful library.

**Pull request merged**